src/electionguard_gui/web/components/shared/key-ceremony-details-component.js

```diff
@@ -6,17 +6,23 @@ export default {
   },
   components: { Spinner },
   data() {
-    return { keyCeremony: null, loading: false };
+    return { keyCeremony: null, loading: false, error: false };
   },
   methods: {
     join: async function () {
       this.loading = true;
       await eel.join_key_ceremony(this.keyCeremonyId)();
       this.loading = false;
     },
-    refresh_key_ceremony: function (keyCeremony) {
-      console.log("key ceremony refreshed", keyCeremony);
-      this.keyCeremony = keyCeremony;
+    refresh_key_ceremony: function (eelMessage) {
+      console.log("key ceremony refreshed", eelMessage);
+      if (eelMessage.success) {
+        this.keyCeremony = eelMessage.result;
+      } else {
+        console.error(eelMessage.message);
+        this.error = true;
```

Comment:
Do we need to set the error to false in the if above this?  since it is in a refresh call, it could be that it is an error first and then it is successful but the error flag will still be true.